queue_job/models/base.py

```diff
@@ -216,3 +216,29 @@ def _job_store_values(self, job):
         :return: dictionary for setting job values.
         """
         return {}
+
+    @api.model
+    def _job_prepare_context_before_enqueue_keys(self):
+        """Keys to keep in context of stored jobs
+
+        Empty by default for backward compatibility.
```

Comment:
```suggestion
```

Reply:
ah yes, well, we'll see if I remove this or the return value, probably the latter